GameOfLifeConsole/Game.cs

```diff
@@ -0,0 +1,116 @@
+﻿using GameOfLifeConsole.Services;
+
+namespace GameOfLifeConsole
+{
+    /// <summary>
+    /// Class of Implementation of GameLogic.
+    /// </summary>
+    public class Game
+    {
+        public GameLogic gameLogic;
+        private FileReadSave _fileReadSave = new FileReadSave();
+        private uint MaxRuns = 50;
+        private int runs = 0;
+        public Game()
+        {
+            gameLogic = new GameLogic();
+        }
+
+        /// <summary>
+        /// The method for running the game.
+        /// </summary>
+        public void Run()
+        {
+            Console.CursorVisible = false;
+            Console.SetCursorPosition(0, 0);
+            while (true)
+            {
+                Console.Clear();
+                AppUserInterface.Start();
+                switch (Console.ReadLine())
+                {
+                    case "M":
+                        {
+                            StartGameManually();
+                        }
+                        break;
+                    case "R":
+                        {
+                            StartGameRandom();
+                        }
+                        break;
+                    default:
+                        {
+                            AppUserInterface.IncorrectDataInput();
+                            Console.ReadLine();
+                        }
+                        break;
+                }
+                while (gameLogic.AliveCells() > 0 && runs++ < MaxRuns)
+                {
+                    string exit;
+                    string save;
+                    Console.Clear();
+                    Console.Title = gameLogic.countIteration.ToString("Iteration {0}");
```

Comment:
I have alive cell counter in class "GameLogic" on line 68.
 Console.WriteLine("Alive cells number: " + count);
Does not it correct?

Reply:
Ah I didn't notice. Currently you have it on pause screen. Would be nice to have opportunity to see the count when game runs in real time, every second it would print alive cell count.

Reply:
Oh I see you have it implemented but the problem is first it prints alive cell count but then you put cursor position on first line and it gets overwritten when you start printing the game.

Reply:
Done. Alive Cells number input in the Title.